tallycash/extension
# Persist keyrings after removing one.

Closes #1489 

### To Test
- Follow the steps in #1489 - removed accounts should properly stay removed on browser close & extension update.
